charts/fybrik/templates/fybrik-admin-role.yaml

```diff
@@ -32,90 +32,5 @@ rules:
   - patch
```

Comment:
admin-role was supposed to be a role applied to a human administrator (to be able to create modules, storage accounts, do debugging, etc.). It currently has nothing to do with it so why not just move whatever is left to the system-role and open an issue for creating a new fybrik-admin-role for humans?